.github/workflows/deploy-apis-to-production.yml

```
@@ -1818,6 +1818,32 @@ jobs:
           kubectl create configmap --dry-run=client -o yaml --from-env-file=.env prod-spatial-api-config | kubectl replace -f - -n production
           gcloud secrets versions access latest --secret="prod-key-device-monitoring-service-account" > google_application_credentials.json
           kubectl create configmap --dry-run=client -o yaml --from-file=google_application_credentials.json prod-spatial-api-config-files | kubectl replace -f - -n production
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)